core/gui/src/app/workspace/component/workspace.component.ts

```diff
@@ -40,6 +40,7 @@ export class WorkspaceComponent implements AfterViewInit, OnInit, OnDestroy {
   public pid?: number = undefined;
   public gitCommitHash: string = Version.raw;
   public showResultPanel: boolean = false;
+  private _onSavingWorkflowObservable: Subject<void> = new Subject();
```

Comment:
we do not use `_` for private variables in typescript.

Reply:
removed